src/app.rs

```diff
@@ -217,8 +224,8 @@ pub struct ApplicationBuilder<'a, 'b, T> {
     pub world: World,
     pool: Arc<ThreadPool>,
     /// Allows to create `RenderSystem`
-    // TODO: Come up with something clever
```

Comment:
Why remove this line?

Reply:
It communicates absolutely 0 useful information.  It's the worst todo I've ever seen.

Reply:
Well, somebody added it there as a remainder to fix something. If we remove that comment, we should 1) fix it or 2) open an issue in case it's a bigger problem.

Reply:
Git blame says @omni-viral wrote this, @omni-viral mind telling us what the intended task for this TODO was?  Maybe I can make it say something more useful.

Reply:
It was about public access to `EventsLoop`. It isn't need to be pub anymore